packages/storefront/src/pages/form-components.md

```diff
@@ -29,6 +29,7 @@ type Variation = {
 
 const variations: Variation[] = [
   { tagName: 'p-checkbox-wrapper', child: '<input type="checkbox" />' },
+  { tagName: 'p-pin-code', child: '', isCustomElement: true },
```

Comment:
Doesn't this define the order of columns/rows? If so, let's stay in sync with the order in VRTs since this component is more similar to text-field-wrapper.